src/nyx/python/nyxus/nyxus.py

```diff
@@ -153,6 +151,48 @@ def featurize(
     
     def using_gpu(self, gpu_on: bool):
         use_gpu(gpu_on)
+
+    def featurize (
+        self,
+        int_fnames: list,
+        seg_fnames: list):
+        """Extract features from image file pairs passed as lists
+
+        Extract stoy
```

Comment:
Copy the long description from `featurize_directory` here and modify accordingly.